extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -517,43 +530,47 @@ angular.module('oppia').component('svgFilenameEditor', {
           ctrl.data.savedSvgFileName.length > 0);
       };
 
+      var getStrokeWidth = function() {
+        var size = ctrl.fabricjsOptions.size;
+        // Removes the word "px" from the end of the string.
+        return parseInt(size.substring(0, size.length - 2));
+      };
+
       ctrl.createRect = function() {
         ctrl.canvas.discardActiveObject();
-        var size = ctrl.fabricjsOptions.size;
         var rect = new fabric.Rect({
-          top: 50,
-          left: 50,
+          top: ctrl.defaultTopCoordinate,
+          left: ctrl.defaultLeftCoordinate,
           width: 60,
           height: 70,
           fill: ctrl.fabricjsOptions.fill,
           stroke: ctrl.fabricjsOptions.stroke,
-          strokeWidth: parseInt(size.substring(0, size.length - 2)),
+          strokeWidth: getStrokeWidth(),
           strokeUniform: true
         });
         ctrl.canvas.add(rect);
       };
 
       ctrl.createLine = function() {
         ctrl.canvas.discardActiveObject();
-        var size = ctrl.fabricjsOptions.size;
-        var line = new fabric.Line([50, 50, 100, 100], {
-          stroke: ctrl.fabricjsOptions.stroke,
-          strokeWidth: parseInt(size.substring(0, size.length - 2)),
-          strokeUniform: true
-        });
+        var line = new fabric.Line(
+          [ctrl.defaultTopCoordinate, ctrl.defaultLeftCoordinate, 100, 100], {
```

Comment:
What do the 100s represent? Maybe assign to and use a descriptive variable name.

Reply:
Done.